x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/upgrade_system_indices/mocks.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SystemIndicesUpgradeStatus } from '../../../../common/types';
+
+export const systemIndicesUpgradeStatus: SystemIndicesUpgradeStatus = {
+  upgrade_status: 'UPGRADE_NEEDED',
+  features: [
+    {
+      feature_name: 'security',
+      minimum_index_version: '7.1.1',
```

Comment:
If we implement my suggestion of asserting against table content we might want to ensure each feature object is unique across all properties (`minimum_index_version`, `index`, `version`), to make it easier to tell which one is which in the table.